packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```
@@ -77,6 +81,15 @@ Conditions:
     Fn::Equals:
       - 'AWS_MANAGED_KEY'
       - Ref: FileAssetsBucketKmsKeyId
+  Untrue:
+    Fn::Equals:
+      - 'CDK_BOOTSTRAP_PERMISSIONS_BOUNDARY'
+      - 'edit this value or remove the condition in order to create the example boundary policy'
```

```suggestion
      - Ref: CreateDefaultPermissionsBoundary
```

And then have another `Parameter`:
```yml
CreateDefaultPermissionsBoundary:
  Default: 'false'
  AllowedValues: ['true', 'false']
  Type: String
```

This way the user can provide the value of the parameter manually in the console if they are using stacksets. But this parameter cannot be populated from the CLI